test/integration/basic_test.go

```diff
@@ -40,20 +44,28 @@ type testCase struct {
 	desc string
 }
 
-func setup(t *testing.T, pfcpAgentConfig string) {
-	providers.RunDockerCommandAttach("pfcpiface",
-		fmt.Sprintf("/bin/pfcpiface -config /config/%s", pfcpAgentConfig))
+func init() {
+	logrus.SetLevel(logrus.TraceLevel)
+}
 
-	// wait for PFCP Agent to initialize, blocking
-	err := waitForPFCPAgentToStart()
-	require.NoErrorf(t, err, "failed to start PFCP Agent: %v", err)
+func setupBESS(t *testing.T, conf pfcpiface.Conf) {
+	// TODO: set up BESS mock
+}
 
-	pfcpClient = pfcpsim.NewPFCPClient("127.0.0.1")
-	err = pfcpClient.ConnectN4("127.0.0.1")
-	require.NoErrorf(t, err, "failed to connect to UPF")
+func teardownBESS(t *testing.T) {
+	// TODO: tear down BESS mock
 }
 
-func teardown(t *testing.T) {
+func setupUP4(t *testing.T, conf pfcpiface.Conf) {
+	err := providers.RunDockerContainer("pfcp-agent", "upf-epc-pfcpiface:0.3.0-dev",
```

Comment:
Is `"upf-epc-pfcpiface:0.3.0-dev"` updated before the test is run? Otherwise we're testing something, but not that PR.